kcp.ts

```diff
@@ -334,7 +334,7 @@ export class Kcp {
 
   /// Receive data from buffer
   recv(buf: Buffer, offset = 0) {
-    offset >>>= 0;
+    offset >>> 0;
```

Comment:
This is an [unsigned right shift assignment][1], not a typo. It coerces `offset` into a 32-bit unsigned integer.

[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unsigned_right_shift_assignment